### ISSUE TITLE
Reduce H.264 latency

### ISSUE THREAD

**Issue by mdevaev** [pikvm/pikvm#784]
TL;DR WebRTC core problem.
* https://webrtc.googlesource.com/src/+/refs/heads/main/docs/native-code/rtp-hdrext/playout-delay
* https://webrtc.googlesource.com/src/+/refs/heads/main/docs/native-code/rtp-hdrext/video-timing/
* https://github.com/Glimesh/janus-ftl-plugin/issues/101
* https://github.com/maitrungduc1410/webrtc/blob/88b8c5280eb75268187c613e66b57b5b9a073899/modules/video_coding/timing/timing.cc
* https://github.com/maitrungduc1410/webrtc/blob/88b8c5280eb75268187c613e66b57b5b9a073899/modules/video_coding/timing/jitter_estimator.cc
* https://stackoverflow.com/questions/71392748/h264-via-webrtc-latency-issue
* https://stackoverflow.com/questions/63220855/webrtc-playoutdelayhint-automaticle-synchronizes-all-tracks
* https://github.com/w3c/webrtc-extensions/pull/15/files
* https://webrtchacks.com/calculate-true-end-to-end-rtt/

**Comment by mdevaev** [comment]
* https://quic.video/blog/replacing-webrtc/
* https://github.com/w3c/webcodecs
* https://developer.mozilla.org/en-US/docs/Web/API/WebCodecs_API

**Comment by mdevaev** [comment]
That's all. All possible optimizations are made in latest updates.

Current latency 35-50ms is lowest possible for all KVMs. https://docs.pikvm.org/latency/
